module/x/gravity/keeper/proposal_handler.go

```diff
@@ -0,0 +1,37 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	distributiontypes "github.com/cosmos/cosmos-sdk/x/distribution/types"
+	"github.com/peggyjv/gravity-bridge/module/x/gravity/types"
+)
+
+func (k Keeper) HandleCommunityPoolEthereumSpendProposal(ctx sdk.Context, p *types.CommunityPoolEthereumSpendProposal) error {
+	// TODO(bolten): is this implicitly called elsewhere?
+	if err := p.ValidateBasic(); err != nil {
```

Comment:
This should be unecessary because this called here

https://github.com/cosmos/cosmos-sdk/blob/9f543b1c3b3d51e8f310fa9ca848d6b101c93f4c/x/gov/keeper/proposal.go#L28-L31